packages/headless/doc-parser/use-cases/commerce.ts

```diff
@@ -2,9 +2,133 @@ import {ActionLoaderConfiguration} from '../src/headless-export-resolvers/action
 import {ControllerConfiguration} from '../src/headless-export-resolvers/controller-resolver';
 import {EngineConfiguration} from '../src/headless-export-resolvers/engine-resolver';
 
-const controllers: ControllerConfiguration[] = [];
+const controllers: ControllerConfiguration[] = [
+  {
+    initializer: 'buildContext',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildCart',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListing',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingBreadcrumbManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingCategoryFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingDateFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingFacetGenerator',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingNumericFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingRegularFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingPagination',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingParameterManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingSort',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductListingUrlManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildProductView',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildRecommendations',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearch',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchBreadcrumbManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchCategoryFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchDateFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchFacetGenerator',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchNumericFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchRegularFacet',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchPagination',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchParameterManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchSort',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchUrlManager',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildSearchBox',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildRelevanceSortCriterion',
+    samplePaths: {},
+  },
+  {
+    initializer: 'buildFieldsSortCriterion',
+    samplePaths: {},
+  },
+];
 
-const actionLoaders: ActionLoaderConfiguration[] = [];
+const actionLoaders: ActionLoaderConfiguration[] = [
+  {
+    initializer: 'loadProductListingActions',
+  },
+  {
+    initializer: 'loadProductListingAnalyticsActions',
```

Comment:
We can remove the product listing analytics actions: they're not being used (nor should they be). In fact, I'll delete those soon 😄